SDPi_Supplement/asciidoc/volume2/dev-28/tf2-dev-28.adoc

```diff
@@ -0,0 +1,63 @@
+:var_transaction_id: DEV-28
+
+:var_label_dev_28_message_contextreport: Notification
+
+
+[#vol2_clause_dev_28]
+=== Notify Change in System Context and Capabilities [{var_transaction_id}]
+
+==== Scope
+
+include::tf2-dev-28-summary.adoc[]
+
+
+==== Actor Roles

```

Comment:
For some reason, "Actor Roles" is not rendered correctly. I couldn't figure out the issue yet... Looks like it doesn't like the the summary include